### PR TITLE
Fixed issue for native where TextFields have a thin border on right edge

### PR DESCRIPTION
**Pull request by xavierstampslafont**

### CODE REVIEW

File: openfl/_internal/renderer/cairo/CairoTextField.hx [hunk 1]
```diff
@@ -33,6 +33,11 @@ class CairoTextField {
 		
 		var textEngine = textField.__textEngine;
 		var bounds = textEngine.bounds;
```
Comment (cactusanctuary) [inline openfl/_internal/renderer/cairo/CairoTextField.hx]: I was a little unsure if I should actually `clone()` the `textEngine.bounds`, but it renders fine. So I think it's OK.

**Comment by jgranick** [comment]
Do you have any test code I can use to see the before and after?

Thank you :grin:

**Comment by xavierstampslafont** [comment]
Hi @jgranick sorry for the long, long silence, my computer broke down a few months back and I've been utterly unable to do any work in my spare time. I don't have anything at the moment to show the issue and how it looks after this fix, unfortunately :(

**Comment by jgranick** [comment]
I think this should be resolved, but please let me know again if there still seems to be an issue. Thank you so much!

**Pull request closed without merging**